src/units.rs

```diff
@@ -19,6 +19,8 @@ pub enum UnitType {
   Mass,
   /// A unit of digital storage, for example [`Kilobyte`]
   DigitalStorage,
+  /// A unit of digital data rate, for example [`KilobytesPerSecond`]
+  DigitalDataRate,
```

Comment:
I'd prefer if it was named `DataTransferRate`, as it's called on Google and Wikipedia

Reply:
To confirm, it's supposed to be `DigitalDataTransferRate`?

Reply:
No, just `DataTransferRate`